Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -618,7 +619,8 @@ namespace AzToolsFramework
         AZ::EntitySystemBus::Handler::BusDisconnect();
         EditorEntityContextNotificationBus::Handler::BusDisconnect();
         AzToolsFramework::ComponentModeFramework::EditorComponentModeNotificationBus::Handler::BusDisconnect();
-
+        ViewportEditorModeNotificationsBus::Handler::BusDisconnect();
```

Comment:
uber nit: This is pretty minor but we usually try and connect/disconnect in the reverse order, something to keep in mind but in practice it likely won't cause problems (in _theory_ it could though 😝)

Reply:
good catch! i'll bundle that trivial change into another PR so i don't have to trigger another AR run 👍